Dockerfile.assisted-service

```diff
@@ -1,10 +1,36 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as certs
+# Generate python client
+FROM swaggerapi/swagger-codegen-cli:2.4.15 as swagger_py
 
+COPY swagger.yaml .
+COPY tools/generate_python_client.sh .
+RUN chmod +x ./generate_python_client.sh && SWAGGER_FILE=swagger.yaml OUTPUT=/build ./generate_python_client.sh
+
+# Generate go client
+FROM quay.io/goswagger/swagger:v0.25.0 as swagger_go
+COPY . .
+RUN mkdir /build
+RUN swagger generate server --template=stratoscale -f swagger.yaml --template-dir=/templates/contrib \
```

Comment:
i think that you need to copy the template dirs as well.

Reply:
Works without copying